test/units/plugins/action/test_synchronize.py

```diff
@@ -99,11 +99,11 @@ class SynchronizeTester(object):
     execute_called = False
 
 
-    def _execute_module(self, module_name, task_vars=None):
+    def _execute_module(self, module_name, module_args=None, task_vars=None):
```

Comment:
Agreed with @cboylan here.  This is the pre-existing way that the test code works.  If we want to change it to have a more complete signature we should probably do that as a separate change.